include/swift/Runtime/Portability.h

```diff
@@ -28,7 +28,10 @@ size_t _swift_strlcpy(char *dst, const char *src, size_t maxlen);
 #ifdef SWIFT_RUNTIME_ATTRIBUTE_ALWAYS_INLINE
 SWIFT_RUNTIME_ATTRIBUTE_ALWAYS_INLINE
 #endif
-__attribute((__format__(__printf__, 2, 3))) inline static int
+#ifdef __clang__
```

Comment:
Per `Config.h`, couldn't you just check `__has_attribute(format)`?

Reply:
Yeah, we could, but at that point, its probably better to introduce a bigger wrapper like the `SWIFT_RUNTIME_ATTRIBUTE_ALWAYS_INLINE`.